src/program/main.cpp

```diff
@@ -260,6 +260,9 @@ int main(int argc, char **argv)
         }
     }
 
+    /* Randomize socket filename */
+    context.socket_filename = tmpnam(NULL);
```

Comment:
Maybe `nullptr` instead of `NULL`? We aren't using C, after all.

Reply:
Misa just recommended that on Discord, I've changed it now.

Reply:
Yeah... I'm Misa.